### PR TITLE
fix(ui): edit alert and misc feedback for alerts page

### PR DESCRIPTION
**Pull request by chirag-madlani**
### Describe your changes :

I worked on the below points

- Edit alerts should work as expected
- View alert should show all the details
- Delete should only have the option to Permanently Delete alerts
- Fix UUID was visible on the UI
- Update AlertActionType label and icons
- Remove ActivityFeed from AlertActionType
- Support AlertActionType -- Email

#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
Screenshot 2022-12-18 at 8 34 24 PM

Screenshot 2022-12-18 at 8 34 36 PM
Screenshot 2022-12-18 at 8 34 49 PM

Screenshot 2022-12-19 at 8 27 39 PM
Screenshot 2022-12-19 at 8 27 35 PM

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx [hunk 1]
```diff
@@ -313,37 +390,169 @@ const AddAlertPage = () => {
...
+                </Form.Item>
+                <Form.Item
+                  label={t('label.receiver-plural')}
+                  labelCol={{ span: 24 }}
+                  name={[name, 'alertActionConfig', 'receivers']}>
+                  <Select
+                    showSearch
+                    mode="tags"
+                    open={false}
+                    placeholder="List emails to subscribe to notifications"
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx:465]: localization

File: openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx [hunk 2]
```diff
@@ -375,277 +585,250 @@ const AddAlertPage = () => {
...
+                      {t('label.save')}
+                    </Button>
+                  </Col>
+                </Row>
+              </Form.Item>
             </Card>
           </Form>
         </Col>
+        <Col span={24} />
+        <Col span={24} />
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx:831]: is it intentional to add 2 empty `<Col span={24} />`?

File: openmetadata-ui/src/main/resources/ui/src/pages/AlertsPage/AlertRowExpanded.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  const [loading, setLoading] = useState(true);
+  const { t } = useTranslation();
+  const fetchAlertActions = useCallback(async () => {
+    try {
+      setLoading(true);
+      const response = await getAlertActionForAlerts(alert.id);
+      setAlertActions(response);
+    } finally {
+      setLoading(false);
+    }
```
Comment (ShaileshParmar11) [inline openmetadata-ui/src/main/resources/ui/src/pages/AlertsPage/AlertRowExpanded.tsx:34]: Don't we need a catch block here?

**Review by ShaileshParmar11 — DISMISSED** [review]

File: openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx (hunk 2 above)
Comment (chirag-madlani) [reply openmetadata-ui/src/main/resources/ui/src/pages/AddAlertPage/AddAlertPage.tsx:831]: Yes to avoid overlap of save button with Slack chat icon

**Review by ShaileshParmar11 — APPROVED** [review]

**Pull request merged**